scripts/train.py

```diff
@@ -45,6 +45,11 @@ def create_models(num_classes, weights='imagenet', multi_gpu=0):
     # Keras recommends initialising a multi-gpu model on the CPU to ease weight sharing, and to prevent OOM errors.
     # optionally wrap in a parallel model
     if args.multi_gpu > 1:
+
+        if args.batch_size < args.multi_gpu:
+            raise ValueError("Batch-Size({}) must be equal or higher than number of GPUs({})".format(args.batch_size,
```

Comment:
Ah I seem my review was outdated. Still though, could you change the message to : 
```python
'Batch size ({}) must be equal to or higher than the number of GPUs ({})'
```
to keep it in line with the other messages?

Reply:
Can do :) I've messaged you on the Slack channel regarding design choices for the use of args in this context.